### PR TITLE
Fixed exception when repeatly choose the same interface

### PR DESCRIPTION
**Pull request by anakin1028**
@blackHatMonkey I found that the command below will pass the check in  `is_valid_interface`:

`./wifiphisher -aI wlan0 -jI wlan0 -p oauth-login`

I also write a test case for this case.

### CODE REVIEW

**Comment by sophron** [comment]
Looks good to me.

File: wifiphisher/common/interfaces.py [hunk 1]
```diff
@@ -383,6 +383,11 @@ def is_interface_valid(self, interface_name, mode=None):
                 raise InvalidInterfaceError(interface_name, mode)
             elif mode == "AP" and not interface_adapter.has_ap_mode:
                 raise InvalidInterfaceError(interface_name, mode)
+
+            # raise an error if interface is already in the _active set
+            if interface_name in self._active:
+                raise InvalidInterfaceError(interface_name)
```
Comment (blackHatMonkey) [inline wifiphisher/common/interfaces.py]: @anakin1028 Can you add the exception to the `docstring` of the method. Also can you add a warning about the fact that a valid interface name will be added to the active list.

**Comment by anakin1028** [comment]
@blackHatMonkey thanks for the suggestion! I add the warning message before raising the exception also add comment in the docstring.

**Comment by blackHatMonkey** [comment]
@anakin1028 I didn't mean to print the warning to the user but rather add to the `docstring` so it's well documented. The user shouldn't be exposed to this kind of information as it is useless to them.

**Comment by anakin1028** [comment]
@blackHatMonkey I got it and fixed that too : )

**Comment by blackHatMonkey** [comment]
Good work @anakin1028 👍 😄 .

**Pull request merged**